test/user_test.rb

```diff
@@ -64,4 +64,32 @@ def test_user_has_organizers
       assert_kind_of Chiketto::Organizer, @user.organizers.first
     end
   end
+
+  def test_user_has_contact_lists
+    find_me
+
+    VCR.use_cassette 'user-contact-lists' do
+      assert_kind_of Chiketto::ContactList, @user.contact_lists.first
+    end
+  end
+
+  def test_user_find_contact_list_returns_contact_list
+    find_me
+
+    VCR.use_cassette 'user-find-contact-list' do
+      assert_kind_of Chiketto::ContactList, @user.find_contact_list(411987)
```

Comment:
If either of you can come up with an alternative way to store the USER_ID and CONTACT_ID that would make it easier to run the test suite I'd love to hear it; I've considered an ENV var but it seems like it might obscure where the test input is coming from.